### DISCUSSION TITLE
Cannot run update query. Result in: Query Error

### DISCUSSION THREAD

**Discussion by Shokhrukh99** [Budibase/budibase#11289] [Help]
Hi all,

I started using Budibase recently. I used it with MS SSQL and found the platform very easy to grasp. I started a project based on Firebase Firestore. Successfully connected to my Firebase project. Easily created and ran "create" and "read" queries. However, "update" queries are not working. 
![image](https://github.com/Budibase/budibase/assets/32006946/9f9f6d07-bf09-4998-9501-5a37e4d113bb)

Here are the fields:
![image](https://github.com/Budibase/budibase/assets/32006946/4710bfcc-c6c4-4e68-9606-631256787808)

![image](https://github.com/Budibase/budibase/assets/32006946/5894165d-5b0e-463f-9f2d-4f51d3f5de19)

I followed the instructions in the documentation. 
I will appreciate any help. Thanks in advance.

Kindest regards,
Shokhrukh

**Comment by melohagan** [comment]
Hey @Shokhrukh99 

Sorry for the late response - I think the problem here is that you simply are missing the *Collection*

**Comment by Shokhrukh99** [comment]
![image](https://github.com/Budibase/budibase/assets/32006946/c500d803-9264-44d5-bfd7-e7e26fa5ff80)

I don't know why it did not show up there, but even with the Collection name typed correctly the issue still persists

**Reply by melohagan** [reply]
Perhaps you need to specify a document in the collection field as well, as shown here: https://docs.budibase.com/docs/firestore#sub-collection
